### DISCUSSION TITLE
Design Notes - naming embedding fields

### DISCUSSION THREAD

**Discussion by SuzanneHunt-KC** [naccdata/uniform-data-set#65] [Help Needed]
Do we need a convention for naming REDCap fields that use embedding so that question layout matches the paper form?  For example, on form A4a, question 2 has many sub-questions and is laid out in a table.  I named my descriptive field with embedding for question  #2 as a4a_2_embed.
@mcleod42

**Comment by mcleod42** [comment]
Hi Suzanne - according to the design notes "The variable name is the name of the form and the section, e.g. a3_section1." Does that work for you?
